src/client/actions/GuildEmojisUpdate.js

```diff
@@ -5,7 +5,7 @@ const Action = require('./Action');
 class GuildEmojisUpdateAction extends Action {
   handle(data) {
     const guild = this.client.guilds.cache.get(data.guild_id);
-    if (!guild || !guild.emojis) return;
+    if (!guild?.emojis) return;
```

Comment:
When would [Guild#emojis](https://discord.js.org/#/docs/main/stable/class/Guild?scrollTo=emojis) be falsy?